android/app/build.gradle

```diff
@@ -160,14 +160,6 @@ android {
             keyAlias 'androiddebugkey'
             keyPassword 'android'
         }
-        release {
```

Comment:
these shouldnt be removed only exposed secrets

Reply:
but the content inside this were keys. Should it be an empty object?
![Screen Shot 2020-05-05 at 12 26 28 PM](https://user-images.githubusercontent.com/59056508/81083997-af5f8700-8ecb-11ea-8059-cfe249828ada.png)